<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>welcomescreen</class>
 <widget class="QDialog" name="welcomescreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TrustedTest</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>App.png</normaloff>App.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(214, 238, 205);</string>
  </property>
  <widget class="QWidget" name="welcome" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1001</width>
     <height>601</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1001</width>
      <height>720</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Back_1.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>170</y>
      <width>361</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:40px;
background-color: rgb(189, 238, 238);</string>
    </property>
    <widget class="QPushButton" name="but_register">
     <property name="geometry">
      <rect>
       <x>205</x>
       <y>130</y>
       <width>141</width>
       <height>38</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color: rgb(255, 255, 255);
border-radius:10px;
}
QPushButton:hover{
border-radius:10px;
background-color: rgb(214, 238, 205);
}</string>
     </property>
     <property name="text">
      <string>Sign Up</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>34</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 34pt &quot;Calibri&quot;;</string>
     </property>
     <property name="text">
      <string>Welcome</string>
     </property>
    </widget>
    <widget class="QPushButton" name="but_login">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>130</y>
       <width>141</width>
       <height>38</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color: rgb(255, 255, 255);
border-radius:10px;
}
QPushButton:hover{
border-radius:10px;
background-color: rgb(214, 238, 205);
}</string>
     </property>
     <property name="text">
      <string>Sign In</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
